src/app/carbon-estimator-form/carbon-estimator-form.component.html

```diff
@@ -235,9 +235,13 @@
 
   <div class="tce-flex tce-gap-4 tce-flex-row-reverse">
     <button
-      class="tce-button-calculate tce-px-3 tce-py-2 disabled:tce-opacity-50 disabled:tce-cursor-not-allowed"
-      type="submit"
-      [disabled]="!estimatorForm.valid">
+      class="{{
```

Comment:
Might be cleaner to use a combination of class and ngClass for the conditional parts.

Reply:
Done :)